pkg/config/images.go

```diff
@@ -19,16 +18,16 @@ func openShiftVersion(imageVersion string) (string, error) {
 	return fmt.Sprintf("v%s.%s.%s", parts[0][:1], parts[0][1:], parts[1]), nil
 }
 
-func selectNodeImage(cs *api.OpenShiftManagedCluster, deployOS string) {
+func selectNodeImage(cs *api.OpenShiftManagedCluster, testConfig *api.TestConfig) {
```

Comment:
passing TestConfig all over the place is pretty horrible.  Ideally I think we should make these functions methods on a new struct type which includes TestConfig, so we don't have to pass it anywhere any more.  Either do that as a new commit in this PR, or open a follow-up to do it (you could assign it to yourself).

Reply:
@thekad please can you do this as a new commit in this PR.  Make the store the *plugin* config in the new struct type, that way @charlesakalugwu will be able to use it for implementing #332 as well.